src/main/java/crystal/Parser.java

```diff
@@ -0,0 +1,75 @@
+package crystal;
+
+import crystal.command.Command;
+import crystal.command.ListCommand;
+import crystal.command.UnmarkCommand;
+import crystal.command.MarkCommand;
+import crystal.command.TodoCommand;
+import crystal.command.DeadlineCommand;
+import crystal.command.EventCommand;
+import crystal.command.ByeCommand;
+import crystal.command.DeleteCommand;
+import crystal.command.FindCommand;
+
+/**
+ * Represents the Parser task.
+ *
+ */
+
+public class Parser {
+
+
+    /**
+     * Returns the respective command object based on the command
+     * the user enters.
+     *
+     * @param command User command
+     * @throws CrystalException When user command is not recognised
+     */
+    public static Command parse(String command) throws CrystalException {
```

Comment:
Also , Have you thought of using switch-case for you're parser? It makes the code so much more neater